File: app/src/main/java/org/techtown/find_gas_station/Data/kakaoResponseModel/kakao/Route.java
package org.techtown.find_gas_station.Data.kakaoResponseModel.kakao;

import com.google.gson.annotations.SerializedName;

public class Route {

    @SerializedName("result_code")
    private int result_code;

    @SerializedName("result_msg")
    private String result_msg;

    private String key;

    private Summary summary;

    public int getResult_code() {
        return result_code;
    }

    public void setResult_code(int result_code) {
        this.result_code = result_code;
    }

    public String getResult_msg() {
        return result_msg;
    }

    public void setResult_msg(String result_msg) {
        this.result_msg = result_msg;
    }

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public Summary getSummary() {
        return summary;
    }

    public void setSummary(Summary summary) {
        this.summary = summary;
    }
}

File: app/src/main/java/org/techtown/find_gas_station/Data/kakao/Response/Route.kt
package org.techtown.find_gas_station.Data.kakao.Response

import com.google.gson.annotations.SerializedName

data class Route(

    @SerializedName("result_code")
    var resultCode : String,

    @SerializedName("result_msg")
    var resultMsg : String,

    @SerializedName("key")
    var key : String,

    @SerializedName("summary")
    var summary : Summary

)